components/SendText.tsx

```diff
@@ -35,24 +36,20 @@ const SendText = ({ org_name, fullAddress, cityStateZip }: SendTextProps) => {
   const postText = async (e: FormEvent): Promise<void> => {
     e.preventDefault()
     try {
-      const validPhoneNumber: RegExp = /^[0-9]{10}$/
-      if (!validPhoneNumber.test(numberToSendTo))
-        throw new Error(activeCopy.error)
+      const validPhoneNumber = validatePhoneNumber(numberToSendTo)
+      if (!validPhoneNumber) throw new Error(INVALID_NUMBER)
 
       const messageToSend = { to: numberToSendTo, message: textToSend }
-      const text = await fetch('/api/QWYJrliMYbKZ9B5wt7EJm1*yD*H%5E66uQ%5E', {
+      const text = await fetch('/api/twilio', {
```

Comment:
ahh yes. I tripped over this one from time to time. Yeah you cant expose those in on github

Reply:
@vsauceda-cfa they weren't exposed on github, they were exposed in the browser when sending a text. what i have done is set up a proxy in the `pages/api` folder. so the message is sent to the next api and the then the next api sends it to twilio. that way, the api creds are never seen by the browser.

it wasn't a huge security vulnerability, but it's better to have it fixed.